open_discussions/settings.py

```diff
@@ -831,6 +831,10 @@ def get_all_config_keys():
 OCW_BASE_URL = get_string("OCW_BASE_URL", "http://ocw.mit.edu/")
 MITX_BASE_URL = get_string("MITX_BASE_URL", "https://www.edx.org/course/")
 MITX_ALT_URL = get_string("MITX_ALT_URL", "https://courses.edx.org/courses/")
+BLACKLISTED_COURSES_URL = get_string(
+    "BLACKLISTED_COURSES_URL",
+    "https://raw.githubusercontent.com/mitodl/open-resource-blacklists/master/courses.txt",
```

Comment:
It looks like this is an empty file? Is it supposed to be populated with course ids over time?

Reply:
It will be populated with course ids in the future, Peter has a [PR open to add the first one](https://github.com/mitodl/open-resource-blacklists/pull/1)